crt_portal/static/sass/_uswds-theme-typography.scss

```diff
@@ -398,16 +398,20 @@ $theme-body-line-height:           5;
 // If true, explicitly style the <body> element with the base styles
 $theme-style-body-element:         false;
 
-// Headings
-$theme-h1-font-size:               '2xl';
-$theme-h2-font-size:               'xl';
-$theme-h3-font-size:               'lg';
+// Headings -- not used
+// NOTE: We don't use these because the design system's font token normalization 
+// changes the size of the font on the fly:
+// https://designsystem.digital.gov/design-tokens/typesetting/overview/
+// Instead, we use REM defined in the next section.
```

Comment:
I HAVE FEELS ABOUT THIS

Reply:
SO MANY FEELS. 

At the risk of being labeled a heretic and cast into a lake of fire, I wonder if it wouldn't be easier to leave the default in for the sake of long-term maintenance. Eventually someone is going to accidentally use, say, the built-in USWDS `h1` mixin instead and fonts will be subtly different as a result. It is difficult to figure out where all of these `theme-*-font-size` variables are used by USWDS, so I don't know the full implications of this change.